ui/Rhino/AGS/dev/AGS_form_move_nodes_cmd.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+
+__commandname__ = "AGS_form_move_nodes"
```

Comment:
you have to decide whether to use `vertices` or `nodes`. in the fix command above, you have used `vertices`...